app/src/main/assets/patient-registration.json

```diff
@@ -62,6 +62,76 @@
           "type": "string",
           "text": "Gender"
         },
+        {
+          "linkId": "PR-telecom",
+          "type": "group",
+          "definition": "http://hl7.org/fhir/StructureDefinition/Patient#Patient.telecom",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-itemContext",
+              "valueExpression": {
+                "language": "application/x-fhir-query",
+                "expression": "ContactPoint",
+                "name": "contactPoint"
+              }
+            }
+          ],
+          "item": [
+            {
+              "linkId": "PR-telecom-system",
+              "definition": "http://hl7.org/fhir/StructureDefinition/Patient#Patient.telecom.system",
+              "type": "string",
+              "text": "system",
+              "initial": [
+                {
+                  "valueString": "phone"
```

Comment:
can we force this to be numeric?

related to https://github.com/OpenSRP/fhircore/issues/46

Reply:
So the function here expects a string https://github.com/hapifhir/org.hl7.fhir.core/blob/e69bad679afdc8382dd309f7341f1c43d96230dc/org.hl7.fhir.r4/src/main/java/org/hl7/fhir/r4/model/ContactPoint.java#L491

Reply:
It's possible to extend but let me wait for reviews here https://github.com/google/android-fhir/pull/326